scripts/developer/tests/basicEntityTest/cylinderSpawner.js

```diff
@@ -0,0 +1,32 @@
+  var orientation = Camera.getOrientation();
+  orientation = Quat.safeEulerAngles(orientation);
+  orientation.x = 0;
+  orientation = Quat.fromVec3Degrees(orientation);
+  var center = Vec3.sum(MyAvatar.position, Vec3.multiply(3, Quat.getForward(orientation)));
+
+  // Math.random ensures no caching of script
+  var SCRIPT_URL = Script.resolvePath("myEntityScript.js")
+
+  var myEntity = Entities.addEntity({
+      type: "Shape",
+      shape: "Cylinder",
```

Comment:
How's that for confusing?  The entity "type" is "Shape".  Its "shape" is "Cylinder".

Similarly, a sphere's "type" is actually "shape" and its "shape" is "sphere"... but we had a SphereEntityItem that pre-existed the ShapeEntityItem whose "type" was "sphere" and whose "shape" was implicit.  So these are the same thing:

```
"type" : "Shape", "shape" : "Sphere"
"type" :  "Sphere"
```

But if you use the former it will be reported back as the latter, were  you to query via script or look at the raw models.json.gz state file saved at the entity-server.